src/JitInterface/src/CorInfoImpl.cs

```diff
@@ -2782,9 +2782,6 @@ private void getCallInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_RESO
 #endif
                             );
                     }
-
```

Comment:
The lines above also look like a bad merge. This code was greatly simplified in the master branch:

https://github.com/dotnet/corert/blob/0444608c0b3c5882f5ca37d50adcd7fe4c0cfe93/src/JitInterface/src/CorInfoImpl.cs#L3284-L3302